source/extensions/common/wasm/wasm.h

```diff
@@ -221,6 +232,26 @@ class Context : public Http::StreamFilter,
            const Http::HeaderMap* response_trailers,
            const StreamInfo::StreamInfo& stream_info) override;
 
+  //
+  // Network::ConnectionCallbacks
+  //
+  void onEvent(Network::ConnectionEvent event) override;
+  void onAboveWriteBufferHighWatermark() override {}
```

Comment:
Shouldn't we be passing these along?

Reply:
No, we're neither buffering nor modifying buffers, so this should be handled by the tcp proxy.